pages/pydata.md

```diff
@@ -0,0 +1,30 @@
+title: PyData Track
+---
+
+This year, for the first time, the teams at PyCon and [PyData London](http://london.pydata.org/) are joining forces to deliver a data-science focussed track at the conference.
+***
+
+## What we are looking for
+This track is mainly focussed on looking for speakers who wish to share their experiences of using Python in data-science. Examples such as:
```

Comment:
This feels a bit wordy -- could you change the opening to "We are looking for speakers..."?